package.json

```diff
@@ -8,7 +8,7 @@
     "lint": "tslint --project .",
     "lint-fix": "tslint --project . --fix",
     "pretest": "yarn lint && tsc --project .",
-    "test": "mocha",
+    "test": "mocha 'test/**/*Test.js'",
```

Comment:
I tried to avoid this, but I didn’t see a good way to do so. I wanted to make running from both the command line and editors (like WebStorm) work as expected with minimal configuration and duplication, and I can’t put globs into `mocha.opts` without forcing those files to always run.